Doc/snackbar.html

```diff
@@ -0,0 +1,124 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../CSS/base.css">
+    <link rel="stylesheet" href="../Doc-css/docs.css">
+    <title>Document</title>
+</head>
+<body>
+    <main class="maple-grid">
+        <!-- navigation section -->
+        <nav class="nav-bar">
+            <div class="nav-left">
+                <button class="open-dialog nav-modal" ><i class="header-text-color fas fa-bars txt-xl navbar-icon "></i></button>
+                <a class="header-text-color txt-xl semibold-font" href="../index.html">Maple UI</a>
+            </div>
+            
+            <div class="nav-right">
+                <a href="" class="navbar-icon txt-lg header-text-color">Docs</a>
+                <a href=""><i class="fab fa-github txt-xl navbar-icon header-text-color"></i></a> 
+            </div>
+        </nav>
+
+        <!-- side section area -->
+
+        <div class="side-section ">
+            <h3>Get Started</h3>
+            <div class="list-wrap">
+                <ul>
+                    <a href="./docs.html"><li class="list-pills">Introduction</li></a>
+                    <a href="./color.html"><li class="list-pills">colors</li></a>
+                    <a href="./typography.html"><li class="list-pills">Typography</li></a>
+                </ul>
+            </div>
+            <h3>Components</h3>
+            <div class="list-wrap">
+                <ul>
+                    <a href="./alert.html"><li class="list-pills">Alert</li></a>
+                    <a href="./avatar.html"><li class="list-pills">Avatar</li></a>
+                    <a href="./badge.html"><li class="list-pills">Badge</li></a>
+                    <a href="./button.html"><li class="list-pills">Button</li></a>
+                    <a href="./card.html"><li class="list-pills">Card</li></a>
+                    <a href="./image.html"><li class="list-pills">Image</li></a>
+                    <a href="./input.html"><li class="list-pills">Input</li></a>
+                    <a href="./text-utilites.html"><li class="list-pills">Text Utilities</li></a>
+                    <a href="./list.html"><li class="list-pills">list</li></a>
+                    <a href="./navigation.html"><li class="list-pills">Navigation</li></a>
+                    <a href="./modal.html"><li class="list-pills">Modal</li></a>
+                    <a href="./rating.html"><li class="list-pills">Rating</li></a>
+                    <a href="./snackbar.html"><li class="list-pills"><h4>Snackbar</h4></li></a>
+                    <a href="./grid.html"><li class="list-pills">Grid</li></a>
+                </ul>
+            </div>
+        </div>
+        <!-- main content area -->
+
+        <div class="main-display ">
+                <h1 class="margin-top-bottom">Snackbar</h1>
+                <p class="simebold-font">This is cute component which don't dis turbs you gently informs you about the message.</p> 
```

Comment:
disturb*
grammar is actually wrong on so many levels-

Reply:
ya sry for inconvenience will correct it